src/ducks/store.jsx

```diff
@@ -0,0 +1,18 @@
+import { legacy_createStore, applyMiddleware } from "redux";
```

Comment:
this file should be in the `src` folder as it doesn't rely to ducks. Ducks are about actions, action creators and reducers